### DISCUSSION TITLE
Presets use the string form of extends

### DISCUSSION THREAD

**Discussion by cjpearson** [renovatebot/renovate#29914] [Request Help] [answered]
### What would you like help with?

I think I found a bug

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

GitLab

### Please tell us more about your question or problem

It looks like many of the presets on the docs site define `extends` as a string. [For example](https://docs.renovatebot.com/presets-group/#groupplaywrightmonorepo).

```
{
  "packageRules": [
    {
      "extends": "monorepo:playwright",
      "groupName": "playwright monorepo",
      "matchUpdateTypes": [
        "digest",
        "patch",
        "minor",
        "major"
      ]
    }
  ]
}
```

It's harmless, but when copying and adapting these rules into our own config, it causes an immediate migration to convert the `extends` property into an array.

**Comment by RahulGautamSingh** [comment, marked as answer]
Will fix this .. thanks for the catch.
Created an issue: https://github.com/renovatebot/renovate/issues/29915
